view/frontend/web/js/view/payment/method-renderer/omise-offsite-mobilebanking-method.js

```diff
@@ -55,6 +95,26 @@ define(
                     }
                 };
             },
+
+            /**
+            * Get a provider list form capabilities api and filter only support type
+            *
+            * @return {Array}
+            */
+            get_available_providers: function () {
+                let _providers = Object.values(window.checkoutConfig.mobile_banking);
+
+                return this.providers.filter((a1) => _providers.find(a2 => {
+                    if (a1.id === a2._id) {
+                        // set currencies from api if is undefined use default value
+                        if (a2?.currencies !== undefined) {
```

Comment:
might be better to use `null` here as null covers both null and undefined cases in javascript

Reply:
Noted 📓  -> https://github.com/omise/omise-magento/pull/337/commits/a0dde384bd84e631ba32d8c03502111eb83c8f84